packages/pangea-sdk/tests/integration/test_vault.py

```diff
@@ -753,23 +800,6 @@ def test_jwt_sym_life_cycle(self) -> None:
             finally:
                 self.vault.delete(item_id=key_id)
 
-    def test_list(self) -> None:
-        list_resp = self.vault.list()
-        assert list_resp.result
-        self.assertGreater(len(list_resp.result.items), 0)
```

Comment:
Don't we want to continue testing Vault's `list` endpoint, though? I know it's called in the new teardown, but I still feel like it deserves its own tests with assertions (ever if only this `assertGreater(len(list_resp.result.items), 0)` one).

Reply:
We have something here https://github.com/pangeacyber/pangea-python/pull/638/files#diff-b7dfd1e6cf296d184ca67a6e47c8528a9b8e853a2cb3069eae644f8949e094acR824:~:text=self.assertEqual(1%2C%20len(list_resp.result.items)) 
Does it work?